app/services/darktide.server.ts

```diff
@@ -448,12 +425,9 @@ let CharacterStoreSchema = z.object({
 							.optional(),
 					}),
 				}),
-				media: z.array(z.unknown()),
 			})
 			.optional()
 	),
-	rerollsThisRotation: z.number(),
-	currentRotationEnd: z.string(),
 })
```

Comment:
Is there a reason to remove the keys you did?

Reply:
Pretty much that we're not using it right this second, A toss up between removing them completely and adding them back in when the functionality is required, or commenting them out for the same

Reply:
Fair enough! I don't mind removing the unused stuff, though I think shop timer should be used